vignettes/opengraph.Rmd

```diff
@@ -0,0 +1,106 @@
+---
+title: "Open Graph"
```

Comment:
Would it be better to make this a little more general and call it page metadata or similar? Then the scope could be anything that appears in `<head>`

Reply:
I agree and wondered about this while working on this. I changed the name of the vignette to `metadata` and I think this is a better fit but am of course open to suggestions. It also raises the question of whether we should change the `opengraph` YAML key to match?

Reply:
I think it's probably fine to keep it as is for now